app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -443,8 +443,8 @@ def parse_service_port_config(port_config, service_id)
         :name        => port_config.name,
         :protocol    => port_config.protocol,
         :port        => port_config.port,
-        :target_port => port_config.targetPort,
-        :node_port   => port_config.nodePort,
+        :target_port => (port_config.targetPort unless port_config.targetPort == 0),
```

Comment:
As I understand from [types.go](https://github.com/moolitayer/kubernetes/blob/master/pkg/api/v1/types.go#L1016)
targetPort port is a string only if it is a name (named port ifrom the pod's container ports list)
otherwise it's an int. the above code works for all these cases:
```
a = 'str'
a = a unless a == 0 # a = 'str'
a = 8080
a = a unless a == 0 # a = 8080
a = 0
a = a unless a == 0 # a = nil
a = nil
a = a unless a == 0 # a = nil
```

Am I missing something? 
ps I could add a test for the above